docs/book/v3/features/error-handling.md

```diff
@@ -213,6 +221,16 @@ class LoggingErrorListenerDelegatorFactory
 }
 ```
 
+Then, enable the delegator factory in your application, such as by adding the following to the `getDependencies()` function in your app or module’s _ConfigProvider.php_ file.
+
+```php
+'delegators' => [
+    ErrorHandler::class => [
+        LoggingErrorListenerDelegatorFactory::class,
+    ]
+],
+```
```

Comment:
I would extend the code example here to make the context clearer.

```suggestion
```php
public function getDependencies(): array
{
    return [
        'factories'  => [
            // …
        ],
        'delegators' => [
            ErrorHandler::class => [
                LoggingErrorListenerDelegatorFactory::class,
            ],
        ],
    ];
}
```
```